terraform/aws/README.md

```diff
@@ -24,11 +24,29 @@ publicly available Amazon machine image (AMI) by default. The AMI can be customi
 through modifications to the [build configuration script](../shared/scripts/setup.sh) 
 and [packer.json](packer.json).
 
-Use the following command to build the AMI:
+
+The packer step has been updated to use HCP packer and an HCL template as opposed to the JSON.
```

Comment:
```suggestion
The packer step has been updated to use HCP Packer and a HCL template as opposed to the JSON.
```

But "has been updated" won't mean anything to a first-time reader once this is merged. I'd leave the whole "we changed something here" out. We probably should include a link for the first mention of HCP Packer as well.

Reply:
right on.  I'll get that done.